command/base_flags.go

```diff
@@ -989,33 +989,3 @@ func (d *timeValue) Get() interface{} { return *d.target }
 func (d *timeValue) String() string   { return (*d.target).String() }
 func (d *timeValue) Example() string  { return "time" }
 func (d *timeValue) Hidden() bool     { return d.hidden }
-
-// -- helpers
```

Comment:
These three functions were unused. I found them as part of my search for `time.ParseDuration`, and thought it better to remove them.